pkgs/dartpad_ui/lib/console.dart

```diff
@@ -118,6 +119,61 @@ class _ConsoleWidgetState extends State<ConsoleWidget> {
                   ],
                 ),
               ),
+              // If there is a JS error, and the app type is Flutter, show a
+              // "File an issue" button. JS errors are normal when executing
+              // Dart (non-Flutter) code.
+              if (widget.output.hasError &&
+                  widget.output.hasJavascriptError &&
+                  appModel.appIsFlutter.value == true)
+                Align(
+                  alignment: Alignment.bottomRight,
+                  child: Row(
+                    mainAxisAlignment: MainAxisAlignment.end,
+                    children: [
+                      Text(
+                        'Oops! Something went wrong.',
+                        style: TextStyle(
+                          color: Theme.of(context).colorScheme.tertiary,
+                        ),
+                      ),
+                      SizedBox(width: 12),
+                      TextButton(
+                        style: ButtonStyle(
+                          backgroundColor: WidgetStateProperty.all(switch (theme
+                              .brightness) {
+                            Brightness.light =>
+                              theme.colorScheme.surface.darker,
+                            _ => theme.colorScheme.surface.lighter,
+                          }),
+                          foregroundColor: WidgetStateProperty.all(
+                            theme.colorScheme.onPrimary,
+                          ),
+                        ),
+                        child: Text('File an issue'),
+                        onPressed: () {
+                          final issueBody = _createGithubIssue(
+                            widget.output.error,
+                          );
+                          const labels = 'type-bug';
+                          final encodedIssueTitle = Uri.encodeComponent(
+                            'Console error',
+                          );
+                          final encodedIssueBody = Uri.encodeComponent(
+                            issueBody,
+                          );
+                          final encodedLabels = Uri.encodeComponent(labels);
+
+                          final String githubIssueUrl =
+                              'https://github.com/dart-lang/dart-pad/issues/new?'
+                              'title=$encodedIssueTitle'
+                              '&body=$encodedIssueBody'
+                              '&labels=$encodedLabels';
+                          launchUrl(Uri.parse(githubIssueUrl));
```

Comment:
If you use the generative (`Uri`) constructor here, the `queryParameters` named param will handle uri encoding automatically.